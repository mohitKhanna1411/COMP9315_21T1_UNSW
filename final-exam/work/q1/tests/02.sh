./pages 2 4 1,2,1,2 <<_Q_
2,8
2,?
?,8
_Q_
