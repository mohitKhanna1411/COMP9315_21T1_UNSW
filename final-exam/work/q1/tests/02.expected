CV = < (1,0) (2,0) (1,1) (2,1) >
query> q[1] = 11100101 11000111 10110001 00101000
q[2] = 00100101 11001000 11110001 10110001
Pages: 2
query> q[1] = 11100101 11000111 10110001 00101000
q[2] = ??
Pages: 0 2 8 10
query> q[1] = ??
q[2] = 00100101 11001000 11110001 10110001
Pages: 2 3 6 7
query> 
